test/regex-compiler.test.ts

```diff
@@ -1,218 +0,0 @@
-const fs = require('fs');
-import { expect } from 'chai';
-const path = require('path')
-const circom_tester = require('circom_tester');
-const generator = require('../compiler/gen')
-const wasm_tester = circom_tester.wasm;
-
-describe("regex compiler tests", function () {
-    [
```

Comment:
Is there a reason these tests were deleted?

Reply:
They are exactly the same as the `js` tests. Keeping the `js` one only since this library is mainly written in `js` anyway.

Reply:
oh we want everything to be in ts if we can; it would be great to keep what's already in ts.